public/app/plugins/datasource/cloudwatch/components/ConfigEditor/ConfigEditor.test.tsx

```diff
@@ -128,178 +129,205 @@ describe('Render', () => {
     datasource.getVariables = jest.fn().mockReturnValue([]);
   });
 
-  it('it should disable access key id field when the datasource has been previously configured', async () => {
-    setup({
-      secureJsonFields: {
-        secretKey: true,
-      },
+  const originalFormFeatureToggleValue = config.featureToggles.awsDatasourcesNewFormStyling;
+
+  const cleanupFeatureToggle = () => {
+    config.featureToggles.awsDatasourcesNewFormStyling = originalFormFeatureToggleValue;
+  };
+
+  function run() {
+    it('it should disable access key id field when the datasource has been previously configured', async () => {
+      setup({
+        secureJsonFields: {
+          secretKey: true,
+        },
+      });
+      await waitFor(async () => expect(screen.getByPlaceholderText('Configured')).toBeDisabled());
     });
-    await waitFor(async () => expect(screen.getByPlaceholderText('Configured')).toBeDisabled());
-  });
 
-  it('should show credentials profile name field', async () => {
-    setup({
-      jsonData: {
-        authType: AwsAuthType.Credentials,
-      },
+    it('should show credentials profile name field', async () => {
+      setup({
+        jsonData: {
+          authType: AwsAuthType.Credentials,
+        },
+      });
+      await waitFor(async () => expect(screen.getByText('Credentials Profile Name')).toBeInTheDocument());
     });
-    await waitFor(async () => expect(screen.getByLabelText('Credentials Profile Name')).toBeInTheDocument());
-  });
 
-  it('should show access key and secret access key fields when the datasource has not been configured before', async () => {
-    setup({
-      jsonData: {
-        authType: AwsAuthType.Keys,
-      },
+    it('should show access key and secret access key fields when the datasource has not been configured before', async () => {
+      setup({
+        jsonData: {
+          authType: AwsAuthType.Keys,
+        },
+      });
+      await waitFor(async () => {
+        expect(screen.getByLabelText('Access Key ID')).toBeInTheDocument();
+        expect(screen.getByLabelText('Secret Access Key')).toBeInTheDocument();
+      });
     });
-    await waitFor(async () => {
-      expect(screen.getByLabelText('Access Key ID')).toBeInTheDocument();
-      expect(screen.getByLabelText('Secret Access Key')).toBeInTheDocument();
+
+    it('should show arn role field', async () => {
+      setup({
+        jsonData: {
+          authType: AwsAuthType.ARN,
+        },
+      });
+      await waitFor(async () => expect(screen.getByText('Assume Role ARN')).toBeInTheDocument());
     });
-  });
 
-  it('should show arn role field', async () => {
-    setup({
-      jsonData: {
-        authType: AwsAuthType.ARN,
-      },
+    it('should display log group selector field', async () => {
+      setup();
+      await waitFor(async () => expect(screen.getByText('Select log groups')).toBeInTheDocument());
```

Comment:
Had to change this from getByLabelText to getByText because the label for Field includes the description too, so the 'Select log groups' text wasn't matching it.